docs/blog/posts/20240606_post_isabelle/index.md

```diff
@@ -0,0 +1,89 @@
+- Alterar mensalmente o conteúdo das variáveis `data_inicio` e `data_fim`  conforme mês vigente da taxação.
+- Alterar mensalmente o conteúdo da variável `data_extenso` conforme o texto a ser colocado na descrição da taxação no terminal Prodemge SISAP.
+- Alterar mensalmente o conteúdo da variável `caminho_documento_taxacao` com o nome da pasta e arquivo da planilha `taxação` atual.
+- Alterar mensalmente o conteúdo da variável `caminho_documento_sisapmasp` com o nome da pasta e arquivo da planilha `base SISAP BO` atual.
+
+## Como funciona? Passo a passo explicado do Automate
+
+**1. Main**
+Este fluxo, ramo principal do robô, ordena os demais subfluxos. 
+- Chama os subfluxos dos tópicos seguintes na ordem em que devem acontecer as etapas.
+
+**2. Subfluxo preparar_planiha**
+- Inicia a planilha `taxação` e executa subfluxo `procv`.
+- Cria nova guia na planilha `taxação`que se chama `lanca_pagamento`. Nessa guia, é formada uma tabela onde cada linha é um professor. Nela, consta o masp, valor a receber das aulas e valor a ser descontado pelo INSS. Esses dois valores são calculados pelo robô.
+ 
+
+  **2.1 Subfluxo procv**
+  - Como a planilha `taxação`não informa o masp dos professores, somente o CPF, esse subfluxo pega o masp que está na planilha `base SISAP BO` e coloca na planilha `taxação`.
+
+**3. Subfluxo taxacao**
+- Baseado na guia criada `lanca_pagamento`na planilha exel `taxacao`, esse subfluxo acessa o terminal Prodemge SISAP e efetua os lançamentos pertinentes.
+- Executa o subfluxo `busca_posicao_admissao`.
+- Executa o subfluxo `liberacao_financeiro`.
+
+  **3.1 Subfluxo busca posição_admissão**
+  - Caso o professor tenha mais de uma admissão, o robô precisa escolher a adimissão correta. Esse subfluxo consiste em fazer a escolha correta da admissão do professor.
+ 
+  **3.2 Subfluxo liberacao_financeiro**
+  - Esse subfluxo insere os valores de INSS e aulas a receber no terminal Prodemge SISAP.
+
+## Utilização do robô
+
+- [x] Conferir se a pasta do mês está com o nome indicado e se estão nela as planilhas `taxação` e `base SISAP BO`. Isso é feito mensalmente.
+- [x] Conferir se as planilhas `taxação` e `base SISAP BO` estão com os nomes apropriados de arquivo. Isso é feito mensalmente.
+- [x] Alterar o conteúdo das variáveis indicadas na seção Premissas desse blog. Isso é feito mensalmente.
+- [x] Nem todos os professores estão registrados na planilha `base SISAP BO`. Para estes professores que não foram cadastrados, o robô não realizará o lançamento do SISAP. Neste caso há duas alternativas:
+  - Fazer a inclusão manual desses professores faltantes na `base SISAP BO`. Isso deve ser feito antes da execução do robô.
+  - Fazer o lançamento manual desses professores no terminal Prodemge SISAP. Isso deve ser feito após a execução do robô. Nesse caso, o robô indica na planilha `taxação`os professores que não foram lançados. Para os que foram lançados ele escreve "lançamento ok". Os que não foram lançados ele deixa a célula vazia.
```

Comment:
@Isabelle-Fernandes, primeiro parabenizar pelo Post, ficou ótimo. Apenas uma dica para aprimorar a formatação do Markdown. Sugiro dar mais um tab nestas opções para que estas duas opções aparecam identadas. Exemplo:

![image](https://github.com/automatiza-mg/automatizacoes/assets/49699290/3c6e7b61-6761-4232-984d-11821ee51747)

Não fiz para você para forçar a utilização da ferramenta.

Reply:
Boa!! Muito obrigada pela dica!